notebook/nbextensions.py

```diff
@@ -299,118 +355,300 @@ def _config_file_name_default(self):
     def install_extensions(self):
         if len(self.extra_args)>1:
```

Comment:
removing this condition work as expected... if we want to warn more than one nbextension we should detect that in another way...